import org.apache.hadoop.fs.PathIOException;
public class FunctionUse14880 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
pathioexception.getTargetPath();
}
}